scripts/install-kubefed.sh

```diff
@@ -86,8 +87,10 @@ elif test X"$LOCATION" = Xolm-kube; then
  ./scripts/kubernetes/olm-install.sh ${OLM_VERSION}
  
  echo "OLM is deployed on kube cluster"
- sed "s,image: quay.*$,image: ${IMAGE_NAME}," -i.bak $CSV_PATH|./hack/catalog.sh | kubectl apply -n $NAMESPACE -f -
- mv $CSV_PATH.bak $CSV_PATH
+ cp $CSV_PATH $CSV_TMP_PATH
+ chmod +w $CSV_TMP_PATH
+ sed "s,image: quay.*$,image: ${IMAGE_NAME}," -i $CSV_TMP_PATH|./hack/catalog.sh $CSV_TMP_PATH | kubectl apply -n $NAMESPACE -f -
```

Comment:
Technically, there is no need to pipe the output of the sed command since we are passing the `CSV_TMP_PATH` now as an argument to the catalog.sh script?
Barring that if you have tested that this works it lgtm.

Reply:
@abhat Yeah, pipe is not necessary, I updated it and tested it. Please help review it again, thank you!